test/bootboot_test.rb

```diff
@@ -8,9 +8,9 @@
 class BootbootTest < Minitest::Test
   def test_does_not_sync_the_gemfile_next_lock_when_unexisting
     write_gemfile do |file, _dir|
-      File.write(file, 'gem "warning"', mode: 'a')
+      File.write(file, 'gem "rake"', mode: 'a')
```

Comment:
`warning` was changed to `rake` just so that the tests could be run on Ruby 2.3 (since warning requires 2.4).